src/Quartz.Examples.AspNetCore/Startup.cs

```diff
@@ -37,17 +37,29 @@ public void ConfigureServices(IServiceCollection services)
                     .SetSchedulerId("Scheduler-Core")
                     .SetSchedulerName("Quartz ASP.NET Core Sample Scheduler");
                 
```

Comment:
// hooks LibLog to Microsoft logging without allowing it to detect concrete implementation
                q.UseQuartzMicrosoftLoggingBridge();

Would be great to just GetService<ILoggerFactory>() and if present, auto-configure MS Loggin, otherwise...
(just going with sensible defaults here)

                q.UseMicrosoftDependencyInjectionJobFactory(); // this should default to MS DI, unless they override?

                q.UseSimpleTypeLoader();

                q.UseInMemoryStore(); // perhaps default to this as well, unless they specify persistent storage?

                q.UseDefaultThreadPool(tp => tp.SetThreadCount(10));

Reply:
What do you mean by GetService, I don't quite follow? Do you mean checking if the MS infrastructure has been registered? If the end user is using say, Serilog, then liblog will sniff it automatically anyway so this is more likely to be effective to ones using MS default console/debug/trace logging or similar. Liblog covers log4net, serilog and nlog which should cover a lot of ground.

Reply:
And with V4 I will definitely switch to MS abstractions and this is a crutch meanwhile.

Reply:
And here many of the switches do have defaults, code is more of a demo of possible switches. As seen in example https://github.com/quartznet/quartznet/blob/master/src/Quartz.Examples/01_SimpleJobScheduler/SimpleJobSchedulerExample.cs#L42-L73 if you don't provide anything the defaults will be used (in memory job store, default thread pool, simple type loader, thread count of 10 which might be all wrong too 🙂 )